drainer/translator/kafka.go

```diff
@@ -126,7 +126,7 @@ func deleteRowToRow(tableInfo *model.TableInfo, raw []byte) (row *obinlog.Row, e
 	columns := tableInfo.Columns
 
 	colsTypeMap := util.ToColumnTypeMap(tableInfo.Columns)
-	columnValues, err := tablecodec.DecodeRow(raw, colsTypeMap, time.Local)
+	columnValues, err := tablecodec.DecodeRow(raw, colsTypeMap, time.UTC)
```

Comment:
This change is not backward compatible, right?

Reply:
yes

Reply:
OK, we should make sure to document the behavior change since the version including this change.

Reply:
don't suggest make such a breaking change for down-type = kafka & file
maybe add a field to indicate the time zone of time.Local ?

Reply:
how to deal with this problem now @leoppro?

Reply:
I will add a flag for `down-type = kafka & file` to avoid breaking compatibility

Reply:
fixed,PTAL again @july2993